exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,3 @@
 const name = " Daniel  ";
-
+let message = name.trim();
 console.log(message);
```

Comment:
You're on the right track here Shima, we need this exercise to output the following string:

```
My name is Daniel and my name is 6 characters long
```

You are currently outputting this:
```
Daniel
```